ush/python/pygfs/task/analysis.py

```diff
@@ -43,12 +43,15 @@ def initialize(self) -> None:
         bias_dict = self.get_bias_dict()
         FileHandler(bias_dict).sync()
 
-        # link jedi executable to run directory
-        self.link_jediexe()
+        # link jedi variational executable to run directory
+        self.link_varexe()
+
+        # link fv3 increment converter executable to run directory
+        self.link_fv3incexe()
```

Comment:
This should be called may be in the `ex-script` or may be create a method say `initialize_fv3inc` and then call through there